### DISCUSSION TITLE
Replacement of WebBrowser1 events in WebView2

### DISCUSSION THREAD

**Discussion by Bhargavi-9** [MicrosoftEdge/WebView2Feedback#3945] [Q&A]
I wanted to move to WebView2 Web Control from WebBrowser2 as the IE is being deprecated. These the some events for which I was facing difficulty to find replacements for:

1. StatusTextChange
2. BeforeNavigate2
3. DocumentComplete
4. WindowClosing
5. NavigateComplete2

I am using RC6.dll for integration of Edge in VB6 code. If anyone finds anything please reply.

**Comment by ProKn1fe** [comment]
- webView.CoreWebView2.NavigationCompleted - CoreWebView2NavigationCompletedEventArgs have WebErrorStatus
- webView.CoreWebView2.NavigationStarting
- webView.CoreWebView2.NavigationCompleted
- webView.CoreWebView2.WindowCloseRequested - maybee
- webView.CoreWebView2.NavigationCompleted
